pack/fhv/fhvc.json

```diff
@@ -261,6 +261,99 @@
+    "traits": "Terror. Power.",
+    "type_code": "treachery"
+  },
+  {
+    "back_flavor": "Bertie approaches, absolutely winded. \"Dr. Marquez requests your immediate presence,\" he huffs, butting into your conversation. \"And believe me, she will not like it if you're late at all.\"",
+    "back_name": "Time to Get to Work!",
+    "back_text": "Each investigator resigns <b>(→R1)</b> <i>(page 9)</i>.",
+    "code": "10681",
+    "doom": 6,
+    "double_sided": true,
+    "encounter_code": "day_of_rest",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "flavor": "The Miskatonic Survey is staying in Hemlock Vale and making preparations for today's expedition. The more the locals trust you, the better you will understand the nature of Hemlock Isle.",
+    "illustrator": "Rob Laskey",
+    "name": "Welcome to Hemlock Vale!",
+    "pack_code": "fhvc",
+    "position": 181,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Forced</b> - When you draw a weakness: cancel its revelation effect and discard it.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "The Brilliance",
+    "back_text": "<b>If you flipped this act while or after playing the Dawn of the First Day prelude, read the following:</b>\n<blockquote><i>The sweetness of the air and the vibrant colors are overwhelming. Your pulse is electric as gooseflesh creeps up your arm and your skin prickles, almost as if under an invisible caress. You are not alone, you never were alone, but the one who is with you is you, and you, and you, and you, and you and you and and and and\nYou catch yourself staring up at the glassy sky, bewildered and drunk with wonder.</i></blockquote>\nYou (and only you) earn 1 bonus experience <i>(do not spend it until instructed)</i>",
+    "clues": null,
+    "code": "10682",
+    "double_sided": true,
+    "encounter_code": "day_of_rest",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "The village brims with excitement! Children play along the roadside wearing animal masks, while others hang gaudy flowers on eaves and along fences.",
+    "illustrator": "Rob Laskey",
+    "name": "Dawn of the First Day",
+    "pack_code": "fhvc",
+    "position": 182,
+    "quantity": 1,
+    "stage": 1,
+    "text": "Investigators cannot take damage/horror or be defeated.\n<b>Objective</b> - Befriend the locals and prepare for the first survey! <i>(Refer to the rules on page 6 for more information on preludes)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "You reconvene back the boarding house. Ms. Olmstead lays out a spread of warm buttered cornbread, roasted radishes the size of your fist, and a pile of grey oily eggs. Your stomach growls as you nibble on some of your field rations rather than partake.",
+    "back_name": "Time to Get to Work!",
+    "back_text": "Check the campaign log:\n<i>If Bertie was rescued,</i> <b>(→R1)</b> <i>(page 42)</i>, \nOtherwise, <b>(→R2)</b> <i>(page 42)</i>",
```

Comment:
extra space before the \n here

Reply:
I think this is missing a trailing Period on both lines as well, after the (page 42).

Reply:
Got it.